### ISSUE TITLE
Cmake support of cryptopp>=6.0 and cryptopp-cmake

### ISSUE THREAD

**Issue by cprieto** [QuantStack/xeus#87]
When trying to build from source I found problems when running `cmake .`

```
CMake Error at CMakeLists.txt:52 (find_package):
  By not providing "Findcryptopp.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "cryptopp",
  but CMake did not find one.

  Could not find a package configuration file provided by "cryptopp" with any
  of the following names:

    cryptoppConfig.cmake
    cryptopp-config.cmake

  Add the installation prefix of "cryptopp" to CMAKE_PREFIX_PATH or set
  "cryptopp_DIR" to a directory containing one of the above files.  If
  "cryptopp" provides a separate development package or SDK, be sure it has
  been installed.
```

 - I downloaded and compiled and installed (successfully) CryptoPP version 7.0.0
 - I tried with both `master` and `0.14.11` versions of Xeus

It makes sense, CyrptoPP 7 does not use CMake so I think it cannot find the cmake config or something.

I tried with an older version of CryptoPP (5.6.5), again, build from source, but this time I get a different message:

```
CMake Error at /opt/cryptopp-CRYPTOPP_5_6_5/cryptopp-config.cmake:1 (include):
  include could not find load file:

    /opt/cryptopp-CRYPTOPP_5_6_5/cryptopp-targets.cmake
Call Stack (most recent call first):
  CMakeLists.txt:52 (find_package)


-- Checking for module 'libzmq>=4.2.3'
--   Found libzmq, version 4.2.5
-- Found LibUUID: /usr/lib/libuuid.so
CMake Error at CMakeLists.txt:156 (get_target_property):
  get_target_property() called with non-existent target "cryptopp-static".


CMake Error at CMakeLists.txt:157 (get_target_property):
  get_target_property() called with non-existent target "cryptopp-static".
```

**Comment by costrouc** [comment]
I have the exact same issue. I am trying to package `xeus` for [nixpkgs](https://github.com/NixOS/nixpkgs)

**Comment by SylvainCorlay** [comment]
Cryptopp stopped supporting cmake with version 6.0.

You can use [cryptopp-cmake](https://github.com/noloader/cryptopp-cmake) instead.

**Comment by SylvainCorlay** [comment]
Totally agree. A PR is welcome if you feel like fixing this now.

**Comment by costrouc** [comment]
Absolutely. I will do that once I have a working nix build of `xeus`.

**Comment by SylvainCorlay** [comment]
Wonderful! I am really excited about the nix packaging and tooling. Don't hesitate to ping us wherever this takes place.

**Comment by martinRenou** [comment]
Sorry @costrouc I didn't see you wanted to do it. I also had problems when installing xeus and fixed it in #94.
